libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/map/MapStyleLoad.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.navigation.dropin.map
+
+import androidx.annotation.IntDef
+import com.mapbox.maps.MapView
+import com.mapbox.navigation.dropin.NavigationView
+
+/**
+ * Map style load parameters
+ */
+object MapStyleLoad {
+
+    /**
+     * Don't load map style. Style loading will be handled by host application.
+     */
+    const val POLICY_NEVER = 0
+
+    /**
+     * Load style only once when [MapView] is added to [NavigationView].
+     */
+    const val POLICY_ONCE = 1
```

Comment:
Why do we need this policy? If user wants to load style only once, won't they simply keep `mapStyleUriDay` and `mapStyleUriNight` unchanged?

Reply:
Convenience.

Reply:
Actually this sounds confusing. When device configuration changes, all views will be recreated and style will be reloaded, even if `POLICY_ONCE` was set.

Reply:
I suggest to remove ONCE policy, and then it can be as simple as a boolean flag that switches between Drop-In loading styles for you and loading them yourself. I don't see why anyone would want to change styles via `ViewOptionsCustomization` without applying it to the map.

Reply:
Sounds good to me. @dzinad Could you update MapViewBinder to use a boolean flag instead?

```kotlin
open val loadMapStyle: Boolean = true // or some name that makes sense
```

Reply:
Done.